include/ts/apidefs.h.in

```diff
@@ -279,7 +287,14 @@ typedef enum {
   TS_HTTP_CACHE_LOOKUP_COMPLETE_HOOK,
   TS_HTTP_PRE_REMAP_HOOK,
   TS_HTTP_POST_REMAP_HOOK,
+  TS_HTTP_REQUEST_BUFFER_READ_COMPLETE_HOOK,
```

Comment:
Need to mark this "incompatible" because this will break plugins compiled for an earlier version.